### ISSUE TITLE
v2 search for C/1995 O1 fails

### ISSUE THREAD

**Issue by mkelley** [Small-Bodies-Node/catch-apis#19]
```
Query NEAT Maui GEODSS from 1996-04-17 to 1998-05-26.
Obtained ephemeris from JPL Horizons.
Critical error: could not search database for this target.
```

**Comment by mkelley** [comment]
Found the problem.  4 NEAT Maui GEODSS images have observation start time == stop time.
* Fixed catch harvester script to detect and correct the problem.
* Added additional debugging info to error message to help in the future.

Reference database is fixed.  To do: make new snapshot files.

**Comment by mkelley** [comment]
@d-w-d This is now fixed in new database backup files, dated 20220208.
